diff --git a/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts b/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts
index dc246ff..8c4cd25 100644
--- a/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts
+++ b/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts
@@ -141,6 +141,7 @@ interface Set$1 extends SetResume$1 {
 		firstEd?: number
 	}
 
+
 	cards: CardList
 
 	boosters?: BoosterList
@@ -425,6 +426,8 @@ declare class CardResume extends Model {
      * @return the full card if available
      */
     getCard(): Promise<Card>;
+
+    updated: string
 }
 
 interface Variants {
@@ -534,6 +537,12 @@ declare class Set extends Model {
     };
     cards: Array<CardResume>;
     boosters?: Array<Booster>;
+
+    totalCount: number;
+
+    abbreviation?: {
+        official?: string;
+    };
     getSerie(): Promise<Serie | null>;
     protected fill(obj: object): void;
 }
