diff --git a/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts b/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts
index dc246ff..f77be23 100644
--- a/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts
+++ b/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts
@@ -343,6 +343,8 @@ interface Card$1<SetType extends SetResume$1 = SetResume$1> extends CardResume$1
 	}
 
 	boosters?: BoosterList
+
+    updated?: string
 }
 
 type StringEndpointList = Array<string>
@@ -692,6 +694,8 @@ declare class Card extends CardResume {
         expanded: boolean;
     };
     boosters?: Array<Booster>;
+
+    updated: string;
     getCard(): Promise<Card>;
     getSet(): Promise<Set>;
 }
diff --git a/node_modules/@tcgdex/sdk/dist/tcgdex.node.mjs b/node_modules/@tcgdex/sdk/dist/tcgdex.node.mjs
index d03e72c..7ae3e2f 100644
--- a/node_modules/@tcgdex/sdk/dist/tcgdex.node.mjs
+++ b/node_modules/@tcgdex/sdk/dist/tcgdex.node.mjs
@@ -512,7 +512,7 @@ var _TCGdex = class _TCGdex {
    */
   encode(str) {
     return encodeURI(
-      str.toString().replace("?", "%3F").normalize("NFC").replace(/["'\u0300-\u036f]/gu, "")
+      str.toString().replace("?", "%3F").normalize("NFC").replace(/["\u0300-\u036f]/gu, "")
     );
   }
 };
